lib/sprockets/sass/v3/importer.rb

```diff
@@ -194,6 +194,8 @@ def filter_all_processors(processors)
         end
 
         def evaluate_path_from_context(context, path, processors)
+          less_processor = Sprockets::Sass::Utils.get_less_class_by_version('LessTemplate')
```

Comment:
We are not processing them, the Less Importer will process them, but we need to call it  : ) 
Problem is the new way of registration of sprockets doesn't allow to fetch engines that are registered on a file extension (e.g. .less) , only if you check the content type ( text/css ) but Sprockets 3.x doesn't have anymore Less support so this content type is now not being used for less. Probablly  i should find a better way of somehow link the content type of text/css to less but I haven't found yet that magic :( 

So as a ugly hack , this is needed . Hope you understand my point..